crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -21,15 +21,17 @@ public Transform Viewpoint
         {
             get
             {
-//#if UNITY_EDITOR
-//                // This does an ok job of grabbing the scene view camera, but the ocean does not render correctly. Seems that maybe the detail center is in the
-//                // wrong place. Not sure..
-//                var sv = SceneView.lastActiveSceneView;
-//                if (sv != null && !EditorApplication.isPlaying && sv.camera != null)
-//                {
-//                    return sv.camera.transform;
-//                }
-//#endif
+#if UNITY_EDITOR
+                // If scene view is focused, use its camera. This code is slightly ropey but seems to work ok enough.
+                if (EditorWindow.focusedWindow.titleContent.text == "Scene")
+                {
+                    var sv = SceneView.lastActiveSceneView;
+                    if (sv != null && !EditorApplication.isPlaying && sv.camera != null)
+                    {
+                        return sv.camera.transform;
```

Comment:
I think making this feature optional would be a good idea, because I would want to be able to inspect the lods from the scene view in some situations based on where the attached viewpoit is located.

Reply:
Yeah that makes sense, made a proposal on latest ,let me know if this works